### ISSUE TITLE
NPM Install Frequently Fails Due to Opencollective

### ISSUE THREAD

**Issue by ericdrobinson** [bootstrap-vue/bootstrap-vue#1885]
# Problem
Running `npm install` (or upgrading `bootstrap-vue` with `npm update bootstrap-vue`) will frequently fail with the following error:

```
npm ERR! path /app/node_modules/opencollective/dist/bin/opencollective.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/app/node_modules/opencollective/dist/bin/opencollective.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```

Google pointed me at [this](https://github.com/opencollective/opencollective-cli/issues/3) but I'm not sure if you've not already implemented the suggestions therein...

## Environment Details
- **Operating system and version:** Docker [NodeJS](https://hub.docker.com/_/node/) Image (Node at version 9.3.0), running on macOS host.
- **NPM Version:** 5.5.1
- **Bootstrap-Vue Version:** 2.0.0-rc.10

**Comment by brightertools** [comment]
Just upgraded to bootstrap-vue 2.0.0-rc.11 and have dependency issues in Visual Studio due to open collective, any idea on a fix?

**Comment by andydevs** [comment]
bootstrap-vue installs just fine on one of my computers but not the other one.
The failing machine is running node 8.11.3 and npm 5.6.0
I'll check my working machine and update this comment with the info

**Comment by crackernutter** [comment]
> bootstrap-vue installs just fine on one of my computers but not the other one.
> I'm trying bootstrap-vue 2.0.0-rc.11
> The failing machine is running node 8.11.3 and npm 5.6.0
> I'll check my working machine and update this comment with the info

My environment is running the same version and is failing with the same issue...

**Comment by pi0** [comment]
Hi. Thanks for your report and patient. We had also lots of problems with open collective for nuxt users. I changed dependency to [nuxt/opencollective](https://github.com/nuxt/opencollective) which should be way more stable. Will be available in versions after 2.0.0-Beta.11
